src/modules/SpaceCalculation.rb

```diff
@@ -578,183 +559,147 @@ def get_partition_info
       # remove the previous failures
       @failed_mounts = []
 
-      # installation stage - Storage:: is definitely present
-      # call Storage::GetTargetMap()
-      targets = Convert.convert(
-        WFM.call("wrapper_storage", ["GetTargetMap"]),
-        :from => "any",
-        :to   => "map <string, map>"
-      )
+      target_partitions = []
+      min_spare = 20 * 1024 * 1024 # minimum free space ( 20 MB )
```

Comment:
The point is that with the new libstorage we need way less nesting to reach the meaningful information in the data structure (4 fewer levels!). When I started with this, I consider it to not be important because I expected a lot of old code to die... but it didn't happen.

I will change this commit to keep the original indentation, even if that breaks our coding standard about indentation. I will add a comment to fix the indentation after TheFinalBigMerge.